.github/ISSUE_TEMPLATE/feature_template.md

```diff
@@ -0,0 +1,51 @@
+# Requesting New Feature
+
+**Name**: *Feature name*
+
+**module**:  *potential_filename.py* 
+
+**package**: */which directory should it go in*
+
+## Scope
+
+*Create a check list using markdowns [checklist syntax](https://help.github.com/en/github/managing-your-work-on-github/about-task-lists) on the scope of the feature*
+
+*What will the feature do?*
+
+*What will the feature not do?* 
+
+## Description
+
+*Please provide a description of the new feature and if will improve or solve a problem.*
+
+### pyDARN Checklist 
+
+-  [ ] Does this fit within pyDARN's scope? 
+-  [ ] Is this a minor change?
+-  [ ] Is this a major change?
+-  [ ] Do you need help developing it? 
+-  [ ] Have you created a pyDARN project to reflect the process of developing it? 
```

Comment:
What does this mean?  I don't understand what this last point is trying to say, and so suggest rewording it.

Reply:
Can you check out the new changes to see if this list is more clearer?

Reply:
It is clearer now.